### ISSUE TITLE
When using Laravel Passport the storage is overridden to /tmp before the keys are loaded

### ISSUE THREAD

**Issue by rockfreak** [brefphp/laravel-bridge#144]
Having migrated from bref v1 to v2 I found an edge case where if using Laravel Passport the storage area is changed to the /tmp directory by laravel bridge before the passport keys have been loaded.

This was disucssed on slack with @mnapoli.

I have solved this for now by adding the following to the `AuthServiceProvider` in the boot function.

`Passport::loadKeysFrom('directory of keys);`

**Comment by georgeboot** [comment]
I personally put the keys in environment variables to work around this issue.

**Comment by mnapoli** [comment]
Yeah this is a breaking change we figured out from v1 to v2. Ideally this should keep working out of the box, I'm not sure yet how to make that happen since we automatically move the entire storage directory to /tmp 🤔
